.github/workflows/semantic-pr-title.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   main:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
     runs-on: ubuntu-latest
     steps:
       - uses: amannn/action-semantic-pull-request@v2.1.0
```

Comment:
There is a version 3.0 https://github.com/marketplace/actions/semantic-pull-request

Reply:
Yup, I had raised an issue yesterday and the version released today https://github.com/amannn/action-semantic-pull-request/issues/72 :)